site/content/tutorial/08-stores/01-writable-stores/text.md

```diff
@@ -24,4 +24,6 @@ Finally, in `Resetter.svelte`, implement `reset`:
 function reset() {
 	count.set(0);
 }
-```
\ No newline at end of file
+```
+
+In the particular case of Single Page Application (SPAs), different routes/pages are usually rendered as components. One could be tempted to use writable stores to keep global application state. Bear in mind, that in practice, reloading the page will reset the stores. 
```

Comment:
"Single Page Application" should probably be "Single Page Applications"

alternatively, if you were saying just "Single Page Application" then you'd add an 'a' and remove an 's' to make it "a Single Page Application (SPA)"